pkgs/tools/security/opensc/default.nix

```diff
@@ -7,33 +7,15 @@
 
 stdenv.mkDerivation rec {
   pname = "opensc";
-  version = "0.19.0";
+  version = "0.20.0";
 
   src = fetchFromGitHub {
     owner = "OpenSC";
     repo = "OpenSC";
-    rev = version;
-    sha256 = "10575gb9l38cskq7swyjp0907wlziyxg4ppq33ndz319dsx69d87";
+    rev = "45e29056ccde422e70ed3585084a7f150c632515";
```

Comment:
Please keep this line unchanged, so reviewers can be sure that the right tag is used.